engine/CMakeLists.txt

```diff
@@ -1368,48 +1389,119 @@ ELSEIF (CMAKE_SYSTEM_NAME STREQUAL Linux)
+                ELSEIF (LSB_LINUX_DISTRIBUTION_CODENAME STREQUAL "hirsute")
+                    IF (USE_PYTHON_3)
+                        SET(CPACK_DEBIAN_PACKAGE_DEPENDS "libpython3.9")
+                    ELSE (USE_PYTHON_3)
+                        MESSAGE(SEND_WARNING "!! Python 2 is not supported on this platform.")
+                        SET(CPACK_DEBIAN_PACKAGE_DEPENDS "")
+                    ENDIF (USE_PYTHON_3)
+                    IF (Boost_DERIVED_VERSION STREQUAL "")
+                        SET(BOOST_VER "1.74.0")
+                    ELSE (Boost_DERIVED_VERSION STREQUAL "")
+                        SET(BOOST_VER "${Boost_DERIVED_VERSION}")
+                    ENDIF (Boost_DERIVED_VERSION STREQUAL "")
+                    SET(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS}, libjpeg62-turbo, libpng16-16, freeglut3, libgtk-3-0, libvorbis0a, libopenal1, libsdl-gfx1.2-5, xdg-utils, libgl1, libopengl0, libglu1-mesa, libboost-atomic${BOOST_VER}, libboost-chrono${BOOST_VER}, libboost-date-time${BOOST_VER}, libboost-filesystem${BOOST_VER}, libboost-log${BOOST_VER}, libboost-python${BOOST_VER}, libboost-regex${BOOST_VER}, libboost-system${BOOST_VER}, libboost-thread${BOOST_VER}")
+                ELSEIF (LSB_LINUX_DISTRIBUTION_CODENAME STREQUAL "ulyana")  # Linux Mint 20 codename "ulyana"
+                    IF (USE_PYTHON_3)
+                        SET(CPACK_DEBIAN_PACKAGE_DEPENDS "libpython3.8")
+                    ELSE (USE_PYTHON_3)
+                        SET(CPACK_DEBIAN_PACKAGE_DEPENDS "libpython2.7")
+                    ENDIF (USE_PYTHON_3)
+                    IF (Boost_DERIVED_VERSION STREQUAL "")
+                        SET(BOOST_VER "1.67.0")
+                    ELSE (Boost_DERIVED_VERSION STREQUAL "")
+                        SET(BOOST_VER "${Boost_DERIVED_VERSION}")
+                    ENDIF (Boost_DERIVED_VERSION STREQUAL "")
+                    SET(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS}, libjpeg62-turbo, libpng16-16, freeglut3, libgtk-3-0, libvorbis0a, libopenal1, libsdl-gfx1.2-5, xdg-utils, libgl1, libopengl0, libglu1-mesa, libboost-atomic${BOOST_VER}, libboost-chrono${BOOST_VER}, libboost-date-time${BOOST_VER}, libboost-filesystem${BOOST_VER}, libboost-log${BOOST_VER}, libboost-python${BOOST_VER}, libboost-regex${BOOST_VER}, libboost-system${BOOST_VER}, libboost-thread${BOOST_VER}")
                 ELSE (LSB_LINUX_DISTRIBUTION_CODENAME STREQUAL "xenial")
                     MESSAGE(SEND_WARNING "!! Unsupported Debian derivative.")
                 ENDIF (LSB_LINUX_DISTRIBUTION_CODENAME STREQUAL "xenial")
 
             ENDIF (USE_DEBIAN_VERSION)
         ENDIF (EXISTS "${CMAKE_BINARY_DIR}/dependency.list")
 
-        # Alternatively, just set a hard-coded set of dependencies...requires more logic for various platforms.
+        # # Alternatively, just set a hard-coded set of dependencies...requires more logic for various platforms.
```

Comment:
Too many comments `#`'s

Reply:
That was intentional. It's supposed to show that the comment on this line no longer applies.

Probably it would be better just to delete the line entirely.

Reply:
yes, if it no longer applies then let's just remove it.

Reply:
Done